src/current/_data/releases.yml

```diff
@@ -6307,3 +6307,31 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v24.2.0-alpha.1
+
+- release_name: v23.2.8
+  lts: true
+  major_version: v23.2
+  release_date: '2024-07-16'
```

Comment:
Might need to update this if it goes out today

Reply:
Just updated thanks - release went out late last night west coast US time